src/content/content.service.ts

```diff
@@ -46,47 +44,46 @@ export class ContentService {
   }
 
   async getOne(collectionName: string, id: string): Promise<any> {
-    return this.gitDBService.getData(collectionName, e => e.id === id);
-  }
-
-  updateMany(
-    collectionName: string,
-    ids: string[],
-    modifier: SetCallback<any>,
-  ): Promise<any> {
-    return this.gitDBService.update(
+    const result = await this.gitDBService.getData(
       collectionName,
-      document => ids.includes(document.id),
-      modifier,
+      e => e.id === id,
     );
+    return result[0];
   }
 
-  async updateOne(
+  async update(
     collectionName: string,
     id: string,
     modifier: SetCallback<any>,
   ): Promise<any> {
-    return this.gitDBService.update(
+    const updatedDocuments = await this.gitDBService.update(
       collectionName,
-      document => document.id === id,
+      document => {
+        return document.id === id;
+      },
       modifier,
     );
+    if (!updatedDocuments.length) {
+      throw { msg: 'File not found' };
```

Comment:
create a new error class that extends general Error. 
```js
class FileNotFoundError extends Error {}
```

then in catch statements following condition:
```js
if (err.msg === 'File not found') {}
```

should rather be:
```js
if (err instanceof FileNotFoundError) {}
```